documentation/tags.md

```diff
@@ -41,7 +42,7 @@ Bootstrap
 Borders
 Box Model
 Box Plot
-Break
+Brain Activity
```

Comment:
```suggestion
Break
```
I don't think Brain Activity is a great tag, as it's only tangential to computer science.
(And you shouldn't be deleting existing tags.)

Reply:
that was a accident I had merge conflict w/ brain activity when I resolved it it got deleted.

Reply:
done